Corgibytes.Freshli.Cli/Functionality/Git/GitArchiveProcess.cs

```diff
@@ -0,0 +1,47 @@
+using System.Diagnostics;
+using System.IO;
+using System.IO.Compression;
+
+namespace Corgibytes.Freshli.Cli.Functionality.Git;
+
+public class GitArchiveProcess : IGitArchiveProcess
+{
+    public string Run(GitSource gitSource, GitCommitIdentifier gitCommitIdentifier, string gitPath, DirectoryInfo cacheDirectory)
+    {
+        // If it exists, make sure to empty it so we are certain we start with a clean slate.
+        var gitSourceTarget = new DirectoryInfo(cacheDirectory.FullName + "/histories" + "/" + gitSource.Hash + "/" + gitCommitIdentifier);
```

Comment:
We should be using `Path.Combine` here:

```suggestion
        var gitSourceTarget = new DirectoryInfo(Path.Combine(cacheDirectory.FullName, "histories", gitSource.Hash, gitCommitIdentifier));
```